src/style/global.js

```diff
@@ -0,0 +1,140 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyles = createGlobalStyle`
+  :root {
+    /* Font settings */
+    --font-family: "Raleway", sans-serif;
+    --font-size-small: 18px;
+    --font-size-medium: 36px;
+    --font-size-large: 48px;
+    --font-size-xlarge: 144px;
```

Comment:
I can see you set some values here, but I also saw many places using sizes directly with `rem`, maybe you should standarize and use only the variables from here?